XCode/Sources/Scopes.swift

```diff
@@ -724,28 +736,28 @@ private func evaluate(_ node: String, _ attrs: [String: String?] = [:], _ closur
     if let placeholder = placeholder { mergedAttributes["placeholder"] = placeholder }
     if let marginheight = marginheight { mergedAttributes["marginheight"] = marginheight }
     if let acceptCharset = acceptCharset { mergedAttributes["accept-charset"] = acceptCharset }
-    
+
     for item in attrs.enumerated() {
         mergedAttributes.updateValue(item.element.1, forKey: item.element.0)
     }
-    
+
     output += mergedAttributes.reduce("") { result, item in
         if let value = item.value {
             return result + " \(item.key)=\"\(value)\""
         } else {
             return result
         }
     }
-    
+
     if let inner = inner {
         scopesBuffer[Process.tid] = output + ">" + (inner) + "</" + node + ">"
     } else {
         let current = scopesBuffer[Process.tid]  ?? ""
         scopesBuffer[Process.tid] = output + ">" + current + "</" + node + ">"
     }
-    
```

Comment:
I think all the extra spaces has something to do with Xcode and your indentation rule maybe?

Reply:
@Vkt0r It's actually extraneous whitespace removed, which should probably be kept, its cleaner and more compatible with others' settings.  Up to you.

Reply:
Yes, I understand your point. But I would like to keep the PR minimal as possible if you don't mind with only the important changes and keep. 

Honestly, I never like this kind of review 😅  so I'll try to add a new rule to Swiftlint ASAP to warning this as a rule and avoid the waste of time on this for future PR.

Can you please keep the indentation to 4 lines as it's in the whole project?

Reply:
I think you misunderstand still, the indentation and number of newlines is unchanged.  The only change is whitespaces/tabs on empty newlines have been removed.  (Check SS for formatting of both)
I'm afraid there's no linter rule to check for this, you can enforce a linter rule to check and warn _for_ extraneous whitespaces and tabs on newlines, but not one to check against it.  If you check for them expecting them to not be there, all PR's will look like this.  I do recommend doing that however and having extraneous whitespace removed (and doing a PR just to remove them so future PR's don't have this "noise")
![image](https://user-images.githubusercontent.com/260871/58192456-cb542380-7c8e-11e9-9419-ce5568b621fe.png)

Reply:
@Vkt0r A different view in sublime with text selected:
Before:
￼![Pasted Graphic](https://user-images.githubusercontent.com/260871/58293573-02602d00-7d94-11e9-8830-2e02601aa767.png)

After:
![Pasted Graphic 1](https://user-images.githubusercontent.com/260871/58293575-04c28700-7d94-11e9-959b-20f356a59257.png)

SwiftLint `trailing_whitespace` will warn for those, it will also autocorrect them all for you

Reply:
Yep, you're right that Swiftlint rule should remove it or warning it! That rule is disabled in our `.swiftlint.yml` so I don't see any problem to enable it and run Swiftlint with autocorrect in another PR to keep this only focused on your changes. 

If you're interested please create another PR removing the `trailing_whitespace` from the excluded rules and run Swiftlint with autocorrect and I'll happily review it.

Thanks again for your contribution!